src/images/bnb-pairs/ata-bnb.svg

```diff
@@ -0,0 +1,77 @@
Fc/IN+yGxz/sZQDVXPYDR7gQw4ZfMGeVSbtsbuW1JPqeka1UA2oCnoG4Am5281FWaoBZQ1NvSd21BHIM4glYdwH/ZDo7rc/Zrv/jDn5qLvb1m9vBJc/+rr1n5/gBbn8TIF8B0ic+9U1gCpmQ4f6Gumkmc9Ac3tMDQAnerBYKG4IgD52CEA/RgiXfluA2ABOeNIAiBLRElSrjArxXWMc6++a55/ft/b17/v39p5i9eY0ZJ00cghZhrFPAk2CO2dmlqVH5x3KITCuqB4esp6Tm1AsaqyrQQAyGKWZgTMkd6Pia5kE7SqfLhgw4niRAP5ENfc4WAQWLnrI1wtuQbI8qK+2OkjEoNzbN1NozrzB5FyklaYpxlBCPFu1TA86b+qRDh4pEyOWNVaxaMJq3kW+UmKEgpx+zAN1bXFF9EEcTDJBMkVdUu96Ff+k2xdBG4QFyR7i4xNeXfZcUM2WKJjWpij0aPKI9baGJE2rPWWJRM5K9HWbpi1bcR77ecdanQl4/0649L3+Hqh7eArOeDHKOZZ/7RBne1BfI+p995Rc/v3nDJThHPeqGvxvJOiHDk6DSDxFNp5z1GbkzpQkDX7MW+XriYxel0YcGOKdmx8wsmdjbJHn/CLFAqJWuJT8rQ/a1ctdjOtBMKyAA2ywAC41VGoQLYGJ2eMcBj+tmFCL8RACQIZdqckaXhHCln/3jKvRYMJGocbTo3FNByWBTSyrQ/KLiGCIKOlNPuC1CBWD05U09pwh7v4Fpsfe48mjX7ENHX+4vmlSSXEfz8DRJQDQ0zqh1RTk0erVj5uRB0QvgwVo8kQNhuZrR/mprnGbkTh/wDwj4IuHPi0jE0FSQPTnpQ/djSc7QtoJLmBWZPzv/9L80Fyvf3v/47VrsfNtOLhyH8mDUmIqDmn9zI/j3yYbLp44mf5xTIQIMbWmBogbvUAsEGUBEskSwJE9u9Odf6eWFfN0AABax2pogV7M0pG1E8/745+cu3zZt/+5bzc3TIj5sZksZWPE0exAMmIy0knUgbAZYlks4LttG8gHw2B2D0vviqNEC1KCWJHAzKla4CpTnEQfFOrAWFg0/5HmRQXCpHi8QT4yVxpEGKC3HVepgW2U38ZjBtnj3S9aCsUQnocJOQwimuMN2rjSNbXZWCRCGAoDxtCGoqnIBALlWBwp7oWmHLWDvRmu2zvDWzRsNRH9LuUsrJu8zysVCJFexryDs2USg/7Y0lGCQLPN+nXPfBYnc6ktgiFLGMKQ8l8u+TT7Q+Vvht9/aTm8O9HXPw4d395w9tBd7mj3LPCP30o2L84gKnq2/V2hfX1/m609/eSweBdMb8g50UEhkyyohw7ajQIx3OHuOfqyM9ShXtnEy3rEL2TKs7AMGQPAHFwWLmUTj7zzghbnReHkZf06TtwBFzFrgBR+TttWepGlPMsSVEfMhmFjBJisnbXCWjNe1MWMMTl9zT9V24lG3OnKYJUvdklDPVY3Ad4jiCPFpGd3SfHmBcCV3LnTfY+nxDnpgXieUlwGNBezjKVc676PL3fiUhBsCyhzrhnNYRgIH4lN+/gnf+6gR+yy4gScpJinuAlvaYo4d19gLyfbDPrkvfMl32HYaQ7wOSBMIof4gmOWwcbWoLxyveW882gYaLoQWGFrgrLSBU9TMhCl3qn1286+ebigAkYF7AVmwBV5cMXE+//Wvz+g9/aA7//OfmjN2KFsenDF7FRTGHn+4I2SQzUI4BNvgCsXKB0ZyV7a5ub1lZrqqURFH6KL6KzeJ4zVRZEAURplGfdJ+5Ul1yOkbfUolh9C6RdrpF5UiD8KaRf+Ajcb0u1MKf565sN36+F6hcrebY0VQRNB8VC48zQndpGTbfKK7XzsSpPqZEM0IA1gk45U5xxvl+4c9e7G7PGfUEZ8mSfJVxBbtd8e2gluTjAMD8INXUTVUDiWUkm+OzSh/CPFKSEWGMZanacLFp61y/9VN/CH9thUu3Ff/Sc7i6uQWe3ex9s+/HEs/fkszTm5MafG9uge0XuQuR4bMvNR0H4rkesdVP7I55b6cCApgCX2XnCvUnSYOOV/qaKHoj/Vs73UBoZ8ydjbaTriF2hjVHp+fXEk52wUADO34qhesCNoBHbGIi2fQsoCkhNNVKIal015Jb88DPoa4j12zt6a4Rgpb+JSl1+mPFysWlCu8ApFPTtdVnp7uaNAAw9T/JOx0dBjhm4dGMoXvLXrt79w+aB79pm6MXF82b78+aN99eNK//PG+OQWPlr0vCuS0bpYAr55cyUAR+tGfn9AyQS75ITrPYiTISWgLar/p0AZM+Sh5s7yjlS2IBNlgobcqBVHj++nVz9t23zdFf/txM799v9ijfBHMeGsnfOMKTFO0C8Cv9zLF5OlwMLTC0wB1rAWHAQaZkp2b6QBT6ueSuhtogF9d2+xCxqC7Bmuz74OzFm+PmxR/+2Hz3f/8/LCz6E6TzBAgXsQsqMgAnsvlIF51pAUCwIsI3AB331YnE0+0zm2YHnVCUGSFopK+AIMTLyMY3ttdinvgPzoJtbmG81u4lg++RO82Bk+5YtKCcLvwcY9IoM1vWz6l4yt3VhuRr62DrIRVdSlBNyxXpLhhSGAEGi7kpRFrBhMjf6XLKEQqrJHPvM3ZAegwxJP01lVrSDqQxAVvdAnO0j07mzkHK0pzy7Tkjb1TMLhSocFgnEiY+daRpXMfE/9RbjF+x7mAt8WznLKTib4Ok1j+BcahoV0bvb3eEzLfUv45V8itsqw7uo1rgdx8T+mOJ58ek/YsN64v83tc2gQzln0B9FEdqvPIiEbHXmY6gI9OZ1VWpUbHA0vn10kfIUpE//DGP5PS7xtQzSgQZMoLsOqIwYofMCFbwgJQtASgVtiW2IWFXCh5IpPOmq9v9c2iGI/vJk5dlBSK4p9syKq4V7U6ZCGxUCYltpuOpn3ZK08UBiZSDMkTiihqAwOk2aVMWG+0xW7L/eNLsP+L8gH2cxFKlCIIu4DS37iSxQnqw0NyTZNg
```

Comment:
What's in there?! Doesn't feel like a safe svg...

Reply:
It's also 409 KB, please optimize image

Reply:
Just my luck 😢(I just used the apeswap farm [source](https://apeswap.finance/images/farms/ata-bnb.svg) image like i saw others had). Hopefully this commit fixes it(I'm not great at making my own images so hopefully this is good enough) https://github.com/beefyfinance/beefy-app/pull/574/commits/5ef987a73fa2bd34644e868d12b2946fff27efca

Reply:
> Your PNG looks good to me. That's pretty a pretty odd svg that apeswap's made. Encoding the png in base64 completely negates any space savings from the SVG elements.

My guess is that they couldn't get the ATA logo to svg format so they added it as a png directly resulting in a weird hybrid and a huge file size. if you run the svg through some optimizers you get [this](https://apeswap.finance/images/farms/blz-bnb.svg) svg(it removes the ata logo part)